### PR TITLE
added meta tags

### PR DESCRIPTION
**Pull request by Kasvit**
**STATUS**
READY

**DESCRIPTION**
add meta tags to account pages and sign in/ login pages
move ".container.box" from partials to an action pages

**CHECK LIST**
- [ ] specs added
- [x] specs passed
- [ ] code coverage >90%
- [x] DB indexes added if needed
- [x] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [x] I've checked new feature as logged in and logged out user if needed
- [ ] "let!" is used in specs

### CODE REVIEW

**Comment by Kasvit** [comment]
image
image
image
image

File: app/views/account/projects/index.html.slim [hunk 1]
```diff
@@ -1,7 +1,8 @@
 .container
   .level
     .level-left
-      .title My projects
+      .title 
+        = title "#{@workspace.name} projects:"
```
Comment (igorkasyanchuk) [inline app/views/account/projects/index.html.slim]: title could be different from this

use title for <title> as a separate call at the top of the view file

File: app/views/account/projects/new.html.slim [hunk 2]
```diff
@@ -1 +1,4 @@
-= render 'form'
+.container.box
+  .title
+    = title "New Project"
```
Comment (igorkasyanchuk) [inline app/views/account/projects/new.html.slim]: and here too
don't mix .title and <title>

File: app/views/account/tasks/edit.html.slim [hunk 3]
```diff
@@ -1 +1,4 @@
-= render 'form'
+.container.box
+  .title
+    = title "Edit Task"
```
Comment (igorkasyanchuk) [inline app/views/account/tasks/edit.html.slim]: everywhere

File: app/views/account/tasks/show.html.slim [hunk 4]
```diff
@@ -1,9 +1,9 @@
 .container
   .content
-    h3 Title
-    = @task.title
+    .title
+      = title "#{@project.name}: #{@task.title}"
     h3 Description
-    = @task.description
+    = description @task.description
```
Comment (igorkasyanchuk) [inline app/views/account/tasks/show.html.slim]: SAME HERE

for description too, don't mix

such values COULD AND WILL BE different

File: app/views/account/workspaces/_form.html.slim [hunk 5]
```diff
@@ -1,3 +1,3 @@
 = simple_form_for [:account, @workspace] do |f|
   = f.input :name, placeholder: 'Name'
-  = f.submit
+  = f.submit
```
Comment (romaslmd) [inline app/views/account/workspaces/_form.html.slim:3]: revert

**Review by igorkasyanchuk — APPROVED** [review]

**Pull request merged**